static/usage/nav/nav-link/angular/page_one_component_ts.md

```diff
@@ -0,0 +1,25 @@
+```ts
+import { Component } from '@angular/core';
+
+import { PageTwoComponent } from './page-two.component';
+
+@Component({
+  selector: 'app-page-one',
+  template: `
+    <ion-header>
+      <ion-toolbar>
+        <ion-title>Page One</ion-title>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content class="ion-padding">
+      <h1>Page One</h1>
+      <ion-nav-link router-direction="forward" [component]="component">
+        <ion-button>Go to Page Two</ion-button>
```

Comment:
This button isn't doing anything when clicked in the Angular Stackblitz, possibly the other buttons too.

Reply:
Looks like a circular dependency?

```
Unhandled Promise rejection:
NG0200: Circular dependency in DI detected for NavController. Find more at https://angular.io/errors/NG0200
```